aws/resource_aws_athena_named_query_test.go

```diff
@@ -98,16 +98,26 @@ resource "aws_s3_bucket" "hoge" {
   force_destroy = true
 }
 
+resource "aws_kms_key" "test" {
```

Comment:
It does not appear this new resource is used in the test configuration and should be removed. 👍